### PR TITLE
Implement Linear API plugin

### PR DESCRIPTION
**Pull request by iNikitaGricenko**
### What changes are being made and why?

- Created Task to retrieve Teams Ids from Linear app
- Created Task to retrieve Labels Ids from Linear app
- Created Task to create issue in Linear app (#2)
---

### How the changes have been QAed?

```yaml
id: myflow
namespace: company.myteam

tasks:
 - id: linear-team
   type: io.kestra.plugin.linear.teams.Search
   token: your_api_token
   teams:
      - first_team_name
      - second_team_name
      
 - id: linear-labels
   type: io.kestra.plugin.linear.labels.Search
   token: your_api_token
   labels:
      - Bug
      - Workflow

  - id: linear-issue
    type: io.kestra.plugin.linear.issues.Create
    token: your_api_token
    teamId: {{ teamId }}
    title: Workflow failed
    description: "{{ execution.id }} has failed on {{ taskrun.startDate }}. See the link below for more details"
    labels:
      - {{ labels.get("Bug") }}
      - {{ labels.get("Workflow") }}
```

---

### Setup Instructions

- https://linear.app/user/settings/api 
- Replace `user` in url to your Linear username
- Create API Key
- Use API Key as token in tasks

### CODE REVIEW

File: src/main/java/io/kestra/plugin/linear/issues/Create.java [hunk 1]
```diff
@@ -0,0 +1,146 @@
...
+        }
+    }
+
+    private String buildInputQuery(
+        String teamId,
+        String title,
+        String description,
+        List<String> labels
+    ) throws JsonProcessingException {
+        ObjectMapper objectMapper = new ObjectMapper();
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/linear/issues/Create.java:113]: Use the ObjectMapper provided by `io.kestra.core.serializers.JacksonMapper.ofJson()`, as Object mappers can be reused across thread we usually define it as a constant.

File: src/main/java/io/kestra/plugin/linear/teams/Search.java [hunk 2]
```diff
@@ -0,0 +1,94 @@
...
+
+    @Schema(
+        title = "Teams names"
+    )
+    @PluginProperty(dynamic = true)
+    private List<String> teamNames;
+
+    @Override
+    public Output run(RunContext runContext) throws Exception {
+        ObjectMapper objectMapper = new ObjectMapper();
```
Comment (loicmathieu) [inline src/main/java/io/kestra/plugin/linear/teams/Search.java:55]: Same here for the object mapper

**Review by loicmathieu — CHANGES_REQUESTED** [review]

**Comment by iNikitaGricenko** [comment]
Sorry, i forgot about new features in java. Will do quick changes. Could you please let me know if i should keep search tasks for teams and labels, or remove them and use one time in Create Issue code?

**Comment by loicmathieu** [comment]
> Could you please let me know if i should keep search tasks for teams and labels, or remove them and use one time in Create Issue code?

The teams.Search and labels.Search tasks are only to find IDs to pass to the Create task?
In those case better to hide that to the user and do everything on a single task

**Comment by iNikitaGricenko** [comment]
> > Could you please let me know if i should keep search tasks for teams and labels, or remove them and use one time in Create Issue code?
> 
> The teams.Search and labels.Search tasks are only to find IDs to pass to the Create task? In those case better to hide that to the user and do everything on a single task

Just thought if those could be useful for users. This is only reason for those tasks

**Review by loicmathieu — APPROVED** [review]
LGTM, thanks

**Pull request merged**